lib/WorseReflection/ReflectorBuilder.php

```diff
@@ -36,6 +36,8 @@ final class ReflectorBuilder
 
     private bool $enableContextualSourceLocation = false;
 
+    private bool $usePhpStanDocblock = true;
```

Comment:
Although I've moved phpstan parser flag away from `ServiceLocator`, it still is hardcoded, as `ReflectorBuilder` is always instantiated with static method, with no access to global config or so.

Reply:
Nevermind. Added as config parameter inside `WorseReflectionExtension` for key `worse_reflection.phpdoc_parser` (options are "native" and "phpstan").